library/x509write_csr.c

```diff
@@ -254,8 +254,8 @@ int mbedtls_x509write_csr_der( mbedtls_x509write_csr *ctx, unsigned char *buf,
         return( MBEDTLS_ERR_ASN1_ALLOC_FAILED );
     }
 
-    len = mbedtls_x509write_csr_der_internal( ctx, buf, size, sig, f_rng,
-                                         p_rng );
+    len = mbedtls_x509write_csr_der_internal( ctx, buf, size, sig,
+                                         f_rng, p_rng );
```

Comment:
I'm sorry if my wording wasn't clear - could you please fix the indentation here?